### DISCUSSION TITLE
Práctica CSS

### DISCUSSION THREAD

**Discussion by chistoperez** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#293] [Show and tell]
Tardé todo el fin de semana pero creo que ya quedó
![screencapture-codepen-io-christopherperez-full-RwjdKdW-2022-03-06-20_56_23](https://user-images.githubusercontent.com/99354481/157054972-0d7f8fe6-e1f0-4bd0-a7c2-8b8a72ebff27.png)

**Comment by Ingrzs** [comment]
Un paso adelante de mi solo me falta el estilo 😅, por cierto en la parte donde das los imagenes abajo, usas contenedores flex o inline-block?

**Reply by chistoperez** [reply]
flex

**Reply by Ingrzs** [reply]
Vale gracias, te quedo 100%👌 , ahora se viene lo bueno con JavaScript

**Comment by GloriaCoralCerecedo** [comment]
Utilizaste boopstrap (?)

**Reply by chistoperez** [reply]
no, puro css

**Reply by GloriaCoralCerecedo** [reply]
Te quedó genial. 

Páginas, videos, atributos y demás que me recomiendes (?)

**Reply by chistoperez** [reply]
usé flexbox para posicionar casi todo, arriba z-index para encimar las imágenes y unos medias para hacerla responsiva a 1080p y a tamaño celular.

**Reply by chistoperez** [reply]
estoy tomando el mini curso de web responsive de freeCodeCamp, tiene videos en youtube en inglés y en español donde explican html, css y js.

**Reply by GloriaCoralCerecedo** [reply]
Oh bueno gracias

**Comment by ChristianDeM** [comment]
Hola que tal. Como le hiciste con los iconos  supongo que son de fontawesome no?
Tengo la duda con el css.  por ejemplo 

`<i class="fa-solid fa-shield">Texto</i>`

como cambio el  color del icono y del texto por separado.

`.fa-solid fa-shield{ color:yellow;}`

ya intente con otra etiqueta dentro.  pero me hace un salto de linea

ejemplo

`<i class="fa-solid fa-shield"><p>Texto</p></i>`

**Reply by chistoperez** [reply]
se puede con 
`<i class="fa fa-shield" style="color:blue"> `
o crear una clase para los colores blue y yellow y metes esa clase en el i
`<i class="fa fa-shield fa_yellow">`
`.fa_yellow { color: blue;}`
`.fa_blue { color: blue;}`
o Saca tu texto del `<i>` así
`<i class="fa-solid fa-shield"></i>  Texto`

**Reply by ChristianDeM** [reply]
Listo.  gracias  opte por sacar el texto. y poner el estilo directo como son solo 2 iconos  de color  amarillo.

**Comment by aleycolen98** [comment]
Hola muy buena, yo apenas tengo el header y el footer me surgió la curiosidad de cómo hiciste  la parte de fecha, lugar y tipo de vacuna, se despliega al hacer click en programar tus vacunas o lo hiciste estático? Podrías explicarme, gracias.

**Reply by chistoperez** [reply]
es un div estático, le di ese posicionamiento con left y top y con z-index hice que este encima de la otra imagen.
